annotationPipeline/src/main/java/org/cbioportal/annotation/pipeline/BatchConfiguration.java

```diff
@@ -71,7 +59,7 @@
     @Autowired
     public StepBuilderFactory stepBuilderFactory;
 
-    @Value("${chunk:10}")
+    @Value("${chunk:1000000}")
```

Comment:
note to self: this chunk size is related to the reader (so not post interval size)